readthedocs/proxito/middleware.py

```diff
@@ -166,6 +167,24 @@ def process_request(self, request):  # noqa
         # Otherwise set the slug on the request
         request.host_project_slug = request.slug = ret
 
+        project = Project.objects.get(slug=request.host_project_slug)
+
+        # This is hacky because Django wants a module for the URLConf,
+        # instead of also accepting string
+        if project.urlconf:
+
+            # Stop Django from caching URLs
+            project_timestamp = project.modified_date.strftime("%Y%m%d.%H%M%S")
+            url_key = f'rtd.urls.fake.{project.slug}.{project_timestamp}'
```

Comment:
```suggestion
            url_key = f'readthedocs.urls.fake.{project.slug}.{project_timestamp}'
```

Just for consistency.